pyproject.toml

```diff
@@ -0,0 +1,26 @@
+[build-system]
+requires = ["setuptools >= 61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "dtspy"
+version = "0.1.0"
+description = "Python client for the Data Transfer Service"
+authors = [
+  { name = "Jeffrey N. Johnson", email = "jeff@cohere-llc.com" },
+]
+readme = "README.md"
+requires-python = ">=3.12"
+dependencies = [
+  "frictionless>=5.17.0,<6",
+  "requests>=2.32.3,<3",
+]
```

Comment:
have you tried running dtspy from a clean py3.12 install? The requirements file is quite long - are these the only two deps? I hope that the testing and doc generation aren't responsible for all the rest!

Reply:
These two are the top-level dependencies, most of the ones listed in the requirements file come from them indirectly.

To see the equivalent `requirements.txt` based on the uv lock file, you can run
`uv pip compile pyproject.toml`
(and maybe add `--no-annotate` to make it easier to diff with current `requirements.txt`)

But yes, there are many that are testing/doc deps...

Reply:
And I'm already using as a dep in a project with
```
[project]
dependencies = [
  "dtspy @ https://github.com/luizirber/dtspy/archive/724f2884e14fee4cc9f9019e3bdf63a46aecf487.tar.gz"
]
```
which points to this PR, and it is all working